tools/topology/topology2/include/components/gain.conf

```diff
@@ -151,7 +159,7 @@ Class.Widget."gain" {
 	uuid 			"A8:A9:BC:61:D0:18:18:4A:8E:7B:26:39:21:98:04:B7"
 	no_pm			"true"
 	cpc 			10183
-	curve_type		"fade"
+	curve_type		"windows_fade"
```

Comment:
is it still default as fade, right? how to make it configurable?

Reply:
Yep, but it should be another PR (change default to "linear" and have e.g. in nocodec topologies this set into "windows_fade" to keep it tested).

The intention with this patch is to fix wrong IPC sent to firmware.

Reply:
ok, @singalsu  @ranj063 @kv2019i @lgirdwood, do we have a way to config this curve_type to make it:
1. windows  sof build, set to windoes_fade.
2. linux/chrome build, set to linear.

we should have a delta config to do this, since windows is not always same with chrome.

Reply:
This PR keeps operation unchanged with windows fade as default, just re-arranged the enum values.

I'd like to make a separate PR to change the default type to linear and change nocodec topology to use the windows fade curve.